docs/how-to-guides/execute-flow-as-a-function.md

```diff
@@ -0,0 +1,63 @@
+# Execute flow as a function
+
+# Set global configs
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](faq.md#stable-vs-experimental).
+:::
+
+## Overview
+
+Promptflow allows you to load a flow and use it as a function in your code.
+This feature is useful when building a service on top of a flow, reference [here](../../examples/tutorials/flow-deploy/create-service-with-flow/) for a simple example service with flow function consumption.
+
+## Load an invoke the flow function
+
+To use the Flow-as-Function feature, you first need to load a flow using the `load_flow` function.
+Then you can consume the flow object like a function by providing key-value arguments for it.
+
+```python
+f = load_flow("../../flows/standard/web-classification")
+f(url="sample_url")
+```
+
+## Config the flow with context
+
+You can overwrite some flow configs before flow function execution by setting `flow.context`.
+
+### Load flow as a function with in-memory connection override
+
+By providing a connection object to flow context, flow won't need to get connection in execution time, which can save time when for cases where flow function need to be called multiple times.
+
+```python
+f.context = FlowContext(
+    connections={"classify_with_llm": {"connection": connection_object}}
```

Comment:
create this connection_object, or indicate the type, or reference the document of connection object here.

Reply:
Good catch, let me add annotation.